### ISSUE TITLE
NVDA failing to read the titles and times of meetings in Outlooks calendar view

### ISSUE THREAD

**Issue by Ednunp** [nvaccess/nvda#9516]
May also be linked to #9087, but not clear.
Steps to reproduce:
1. Enter the calendar view of Outlook 365.
2. Press tab to move through the days appointments.

The following behavior is raised: NVDA reads the word "meeting" each time tab is pressed.

Expected behavior: NVDA should read the title and time of each meeting or appointment.

Software versions:
Windows version 10
NVDA Alpha 17099
Outlook 365 version 16.0.11601.20068

NVDA is installed on this PC.
Have you tried other versions of VDA? Yes, multiple over the past few weeks, it seems this is a problem introduced in a new build of Outlook.
Is this issue resolved by restarting your PC? no.

**Comment by Adriani90** [comment]
This is probably caused by the way Outlook exposes Information when tabbing. Please try to navigate between appointments with ctrl+dot and ctrl+coma. Do you see the same behavior then?

**Comment by leonardder** [comment]
[**@Adriani90**](https://github.com/Adriani90) commented on [24 Apr 2019, 13:35 CEST](https://github.com/nvaccess/nvda/issues/9516#issuecomment-486185927 "2019-04-24T11:35:39Z - Replied by Github Reply Comments"):
> This is probably caused by the way Outlook exposes Information when tabbing. Please try to navigate between appointments with ctrl+dot and ctrl+coma. Do you see the same behavior then?

I don't think this is true. It sounds to me that NVDA is unable to access Outlooks object model.

Are you explicitly running Outlook as administrator by any chance?

**Comment by Adriani90** [comment]
This is not reproducible in NVDA 2024.2 Beta and Outlook 365 or Outlook 2016. You have to switch the arrangement of your appointments to day view, working week, or week view in calendar Outlook / view / aranging order. Then appointments will be reported properly on tab key. In month view tab key will move from week to week instead. I am closing as works for me.
